src/cdomain/value/cdomains/valueDomain.ml

```diff
@@ -724,7 +724,7 @@ struct
       Array (CArrays.set ask n (array_idx_top) v)
     |                 t , Blob n       -> Blob (Blobs.invalidate_value ask t n)
     |                 _ , Thread tid   -> Thread (Threads.join (Threads.top ()) tid)
-    |                 _ , JmpBuf _     -> state (* TODO: no top jmpbuf *)
+    |                 _ , JmpBuf (b, c, _)     -> JmpBuf(b, c, true)
```

Comment:
The comment should probably still remain, as we normally would have to do more here but just haven't gotten around to it yet.